src/rpc/misc.cpp

```diff
@@ -801,6 +806,7 @@ UniValue getaddressmempool(const JSONRPCRequest& request)
             "    \"timestamp\"  (number) The time the transaction entered the mempool (seconds)\n"
             "    \"prevtxid\"  (string) The previous txid (if spending)\n"
             "    \"prevout\"  (string) The previous transaction output index (if spending)\n"
+            "    \"isInvite\"  (boolean) If transaction is an invite\n"
```

Comment:
should this be 'isinvite' seems inconsistent with other values.

Reply:
thought about it, but here's an `outputIndex` couple lines above
https://github.com/meritlabs/merit/blob/59a402867c267a338f907dc67f4ff8132223f9ea/src/rpc/misc.cpp#L804